front/src/stories/IntervalsEditor.stories.tsx

```diff
@@ -59,28 +59,28 @@ type IntervalsEditorProps = {
   defaultValue: number | string;
   defaultUnit?: string;
   emptyValue: unknown;
-  exampleData: IntervalItem[];
+  data: IntervalItem[];
   fieldLabel: string;
   intervalType: INTERVAL_TYPES.NUMBER | INTERVAL_TYPES.NUMBER_WITH_UNIT | INTERVAL_TYPES.SELECT;
-  toolsConfig?: IntervalsEditorToolsConfig;
+  selectOptions?: string[];
   title?: string;
+  toolsConfig?: IntervalsEditorToolsConfig;
   totalLength: number;
-  selectOptions?: string[];
   units: string[];
 };
 
 const IntervalsEditorWrapper: React.FC<IntervalsEditorProps> = (props) => {
-  const { exampleData, intervalType, selectOptions, totalLength, units = [] } = props;
-  const [data, setData] = useState<IntervalItem[]>(
-    fixLinearMetadataItems(exampleData.filter(notEmpty), totalLength)
+  const { data, intervalType, selectOptions, totalLength, units = [] } = props;
+  const [_data, setData] = useState<IntervalItem[]>(
```

Comment:
the underscore is not very advised, do you have another solution

Reply:
Yes, I can change that :)